src/components/bcros/filing/addStaffFiling/Index.vue

```diff
@@ -62,21 +62,27 @@ const restoreCompany = async (restorationType: FilingSubTypeE = null) => {
 const allActions: ComputedRef<Array<MenuActionItem>> = computed(() => {
   return [
     { // <!-- Registrar Notation -->
-      showButton: isActionVisible(AllowableActionE.REGISTRARS_NOTATION),
+      showButton:
+        isActionVisible(AllowableActionE.REGISTRARS_NOTATION) &&
+        isAuthorized(AuthorizedActionsE.STAFF_FILINGS),
       disabled: !business.isAllowed(AllowableActionE.REGISTRARS_NOTATION),
       datacy: 'registrar-notation',
       label: t('label.filing.staffFilingOptions.registrarsNotation'),
       click: () => { openRegistrarNotationModal.value = true }
     },
     { // <!-- Registrar Order -->
-      showButton: isActionVisible(AllowableActionE.REGISTRARS_ORDER),
+      showButton:
+        isActionVisible(AllowableActionE.REGISTRARS_ORDER) &&
+        isAuthorized(AuthorizedActionsE.STAFF_FILINGS),
       disabled: !business.isAllowed(AllowableActionE.REGISTRARS_ORDER),
       datacy: 'registrar-order',
       label: t('label.filing.staffFilingOptions.registrarsOrder'),
       click: () => { openRegistrarOrderModal.value = true }
     },
     { // <!-- Court Order -->
-      showButton: isActionVisible(AllowableActionE.COURT_ORDER),
+      showButton:
+        isActionVisible(AllowableActionE.COURT_ORDER) &&
+        isAuthorized(AuthorizedActionsE.STAFF_FILINGS),
```

Comment:
Court Order is now separate from STAFF_FILINGS. Use COURT_ORDER_FILING here.

Reason: Filings UI also needs to check if Court Order is authorized, so it's better to keep it separate. Another way of looking at this is that most of the staff filings are in THIS UI and are simple dialogs, so we can group those as STAFF_FILINGS, but the other ones are implemented as full pages in Filings UI or Create UI or Edit UI.